docs/modules/reference/pages/service-assurance/monitors/DNSResolutionMonitor.adoc

```diff
@@ -80,84 +80,93 @@ For Debian/Ubuntu, use `/var/lib/opennms/rrd/response`.
 <service name="DNS-Resolution-CNAME-and-MX" interval="300000" user-defined="false" status="on">
-    <parameter key="retry" value="2"/>
-    <parameter key="timeout" value="2000"/>
-    <parameter key="record-types" value="CNAME,MX"/>
-    <parameter key="lookup" value="www.google.comm"/>
-    <parameter key="rrd-repository" value="/opt/opennms/share/rrd/response"/>
-    <parameter key="rrd-base-name" value="dns-res-cname-mx"/>
-    <parameter key="ds-name" value="dns-res-cname-mx"/>
+    <parameter key="retry" value="2"/> <1>
+    <parameter key="timeout" value="2000"/> <2>
+    <parameter key="record-types" value="CNAME,MX"/> <9>
+    <parameter key="lookup" value="www.google.comm"/> <7>
+    <parameter key="rrd-repository" value="/opt/opennms/share/rrd/response"/> <4>
+    <parameter key="rrd-base-name" value="dns-res-cname-mx"/> <5>
+    <parameter key="ds-name" value="dns-res-cname-mx"/> <6>
 </service>
 
-<monitor service="DNS-Resolution-v4" class-name="org.opennms.netmgt.poller.monitors.DNSResolutionMonitor" /> <1>
-<monitor service="DNS-Resolution-v4-lookup" class-name="org.opennms.netmgt.poller.monitors.DNSResolutionMonitor" /> <1>
-<monitor service="DNS-Resolution-v6" class-name="org.opennms.netmgt.poller.monitors.DNSResolutionMonitor" /> <1>
-<monitor service="DNS-Resolution-Sub" class-name="org.opennms.netmgt.poller.monitors.DNSResolutionMonitor" /> <1>
-<monitor service="DNS-Resolution-v4-and-v6" class-name="org.opennms.netmgt.poller.monitors.DNSResolutionMonitor" /> <1>
-<monitor service="DNS-Resolution-v4-or-v6" class-name="org.opennms.netmgt.poller.monitors.DNSResolutionMonitor" /> <1>
-<monitor service="DNS-Resolution-CNAME-and-MX" class-name="org.opennms.netmgt.poller.monitors.DNSResolutionMonitor" /> <1>
+<monitor service="DNS-Resolution-v4" class-name="org.opennms.netmgt.poller.monitors.DNSResolutionMonitor" /> <10>
+<monitor service="DNS-Resolution-v4-lookup" class-name="org.opennms.netmgt.poller.monitors.DNSResolutionMonitor" /> <10>
+<monitor service="DNS-Resolution-v6" class-name="org.opennms.netmgt.poller.monitors.DNSResolutionMonitor" /> <10>
+<monitor service="DNS-Resolution-Sub" class-name="org.opennms.netmgt.poller.monitors.DNSResolutionMonitor" /> <10>
+<monitor service="DNS-Resolution-v4-and-v6" class-name="org.opennms.netmgt.poller.monitors.DNSResolutionMonitor" /> <10>
+<monitor service="DNS-Resolution-v4-or-v6" class-name="org.opennms.netmgt.poller.monitors.DNSResolutionMonitor" /> <10>
+<monitor service="DNS-Resolution-CNAME-and-MX" class-name="org.opennms.netmgt.poller.monitors.DNSResolutionMonitor" /> <10>
 ----
-<1> Required *monitor* section for each service.
+<1> Number of attempts to test a service’s status
+<2> Timeout for the isReachable method, in milliseconds.
+<3> resolution-type: v4, v6, both (v4 and v6), either (v4 or v6)
+<4> Base directory of an RRD repository in which to store this service monitor’s response-time samples.
+<5> The file name of the RRD file (minus the .rrd or .jrb file extension)
```

Comment:
Markdown isn't happy with the adoc format, the markdown gets semi lost when rendered [no gray background]. This will need to be resolved likely a different way.